drv/spartan7-loader/src/main.rs

```diff
@@ -172,6 +172,8 @@ fn init() -> Result<(), LoaderError> {
     ringbuf_entry!(Trace::WaitForDone);
     loader.finish_bitstream_load()?;
 
+    // Load-bearing delay, do not remove
```

Comment:
Do we still consider this load bearing? If so, can we get a slightly better description what we're waiting for?

Reply:
Yup, added a comment (although we could still fine-tune it further)